BaxterEnv.lua

```diff
@@ -168,8 +168,7 @@ end
 
 -- Min and max reward
 function BaxterEnv:getRewardSpec()
-	return 0, 0.01, -0.015, 0.1, -0.1, 1, -1
-	-- erroreous, correct dir, wrong dir, contact obj, failed pick up, success, timeout
+	return 0, 1, -1, 10, -10
```

Comment:
Can you please add a comment after this line saying what reward is giving for what action?